### ISSUE TITLE
OSX Homebrew lib path for Apple M1

### ISSUE THREAD

**Issue by stepelu** [lilohuang/PyTurboJPEG#61]
On Apple silicon, Homebrew installs under `/opt/homebrew` instead of `/usr/local`.
Consequently, `brew install jpeg-turbo` results in the lib path `"/opt/homebrew/opt/jpeg-turbo/lib/libturbojpeg.dylib"`.
Would it please be possible to check for this path as well on OSX when calling the default constructor `TurboJPEG()`?
As of now the user has to dig into the brew install to figure out the right path to pass to the `TurboJPEG` constructor.

**Comment by lilohuang** [comment]
I'm using apple silicon as well, but previously I installed turbo-jpeg from official release installer (https://sourceforge.net/projects/libjpeg-turbo/files/).

I will update the default path to accommodate home brew installation path in the following days. thanks.   

```shell
lilohuang@MacBook-Pro homebrew % file /opt/homebrew/opt/jpeg-turbo/lib/libturbojpeg.dylib
/opt/homebrew/opt/jpeg-turbo/lib/libturbojpeg.dylib: Mach-O 64-bit dynamically linked shared library arm64
```

**Comment by lilohuang** [comment]
Fixed with v1.6.6 https://github.com/lilohuang/PyTurboJPEG/releases/tag/v1.6.6
